src/core/tests/conditional_compilation/ngraph_cc_collect.cpp

```diff
@@ -24,18 +27,29 @@ TEST(conditional_compilation, collect_op_scope) {
 #define ov_op_Scope0 1
     int n = 0;
 
-    // Simple scope is enabled
+    // Simple Scope0 is enabled
     OV_OP_SCOPE(Scope0);
     n = 42;
     EXPECT_EQ(n, 42);
 
-    // Simple scope is disabled
+    // Simple Scope1 is enabled regardless of macros
     OV_OP_SCOPE(Scope1);
     n = 43;
     EXPECT_EQ(n, 43);
 #undef ov_op_Scope0
 }
 
+TEST(conditional_compilation, collect_ops_in_opset) {
+    ov::OpSet opset("test_opset1");
```

Comment:
I propose to add: `#define ov_opset_test_opset3_Abs 1` in order to check that macro doesn't change behavior

Reply:
It is not needed at all because such macros should be set on SELECTIVE_BUILD=ON stage only. And moreover, there are no defines checks on SELECTIVE_BUILD_ANALYZER=ON stage, so such define is unreachable